app/controllers/resources_controller.rb

```diff
@@ -205,6 +229,16 @@ def signed_id?(file_id)
     ActiveStorage.verifier.valid_message?(file_id)
   end
 
+  def globus_id?(file_id)
+    file_id =~ %r{globus://}
```

Comment:
This will produce a match object which I don't think we care about. Would you consider `file_id.start_with?('globus://')`?

Additionally, I wonder if we want to stick `'globus://'` in a constant up above and use the constant throughout the class.